flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/batch/sql/SqlITCase.java

```diff
@@ -118,4 +119,39 @@ public void testJoin() throws Exception {
 		String expected = "Hi,Hallo\n" + "Hello,Hallo Welt\n" + "Hello world,Hallo Welt\n";
 		compareResultAsText(results, expected);
 	}
+
+
+	@Test
+	public void testLikeWithEscapeFromDataSet() throws Exception {
```

Comment:
Oh, and addd `testNotLikeWithEscape ` , `testNotSimilarWithEscape`  cases. 

I'm not sure whether we should support LIKE ESCAPE for Table API. cc @fhueske  @twalthr  what do you think ?